### DISCUSSION TITLE
Change repository url for all Argo-CD applications

### DISCUSSION THREAD

**Discussion by L-U-C-K-Y** [argoproj/argo-cd#11031] [Q&A] [answered]
Hi all

We have moved our git repository to another domain and I would like to update all existing applications to the new URL.

I have already added the new repositories to Argo-CD.

Is there any way to mass-update all app urls?

Thanks!

**Comment by alexmt** [comment]
I would suggest to script it using `argocd` CLI. Following command updates one Argo CD app:

```
argocd app set guestbook --repo https://github.com/argoproj/argocd-example-apps-UPDATED
```

**Reply by szelga** [reply]
can you change it from helm URL to git URL that way?

**Comment by L-U-C-K-Y** [comment, marked as answer]
@alexmt thanks for your help, I created a script for it here: https://github.com/L-U-C-K-Y/replace-argocd-app-repo-urls

**Comment by jfr992** [comment]
hey guys, out of curiosity, that triggers the recreation of the resources managed by that app?
